app/main/forms/frameworks.py

```diff
@@ -45,6 +45,7 @@ class ReuseDeclarationForm(Form):
     """
     reuse = BooleanField(
         'Do you want to reuse the answers from your earlier declaration?',
+        false_values={False, 'False', 'false', ''},
```

Comment:
The WTForms docs say that `false_values` should be a sequence of strings, so not sure about including the raw `False` in here.
http://wtforms.readthedocs.io/en/latest/fields.html